spec/requests/students_spec.rb

```diff
@@ -48,5 +47,21 @@
         expect(JSON.parse(response.body)).to eq({ 'error' => 'student not found' })
       end
     end
+
+    context 'when a student is found' do
+      it 'reponds with 200 response' do
+        student = create(:student)
+        get student_path(student[:id])
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'responds with object containing student' do
+        student = create(:student)
+        get student_path(student[:id])
+        res_body = JSON.parse(response.body)
+        expect(res_body).to include('name' => student[:name])
```

Comment:
This isn't DRY, it's a pain, and I need to do it for every time I get a student object.  Need to figure out what to do.